packages/graphics/src/Graphics.js

```diff
@@ -1033,7 +1033,16 @@ export default class Graphics extends Container
     _calculateBounds()
     {
         this.finishPoly();
-        const lb = this.geometry.bounds;
+
+        const geometry = this.geometry;
+
+        // skipping when graphics is empty, like a container
+        if (!geometry.graphicsData.length)
+        {
+            return;
+        }
+
+        const lb = geometry.bounds;
```

Comment:
Could you make a tiny quality improvement here by destructuring the bounds props? I know `lb` was there before, but I don't like variable names that are confusing.
```suggestion
        const { minX, minY, maxX, maxY } = geometry.bounds;
```